### PR TITLE
Slackcoder/track show peer branches

### PR DESCRIPTION
**Pull request by slack-coder**
Show all peer branches when listing peers through rad-track.

References #159 for 'rad-track --local should show non-default branches too'

### CODE REVIEW

**Comment by slack-coder** [comment]
@cloudhead should I move get_peer_branches to common:git?  It also looks like get_remote_head isnt used anymore.

**Comment by cloudhead** [comment]
I think it's fine where it is for now. I'd like to keep the `git` module free from dependencies on librad.

File: track/src/lib.rs [hunk 1]
```diff
@@ -283,32 +283,57 @@ pub fn show(
     Ok(())
 }
 
+// return the Peer's branches with their Oid's by parsing references in the storage's git
+// repository.
```
Comment (cloudhead) [inline track/src/lib.rs:287]: Docstrings start with `///` and a capital

File: track/src/lib.rs [hunk 2]
```diff
@@ -283,32 +283,57 @@ pub fn show(
     Ok(())
 }
 
+/// return the Peer's branches with their Oid's by parsing references in the storage's git
```
Comment (cloudhead) [inline track/src/lib.rs:286]:
```suggestion
/// Teturn the Peer's branches with their Oid's by parsing references in the storage's git
```

**Pull request closed without merging**